i18n/it.yaml

```diff
@@ -15,11 +15,17 @@
     other: "{{ .Count }} parole"
 
 - id: toc
-  translation: "Tabella dei Contenuti"
```

Comment:
`Tabella dei Contenuti` fits more for Table of content a/c to google translate, let us keep this as it is

Reply:
> `Tabella dei Contenuti` fits more for Table of content a/c to google translate, let us keep this as it is

I'm sorry, but I'm Italian native and I'm pretty sure I know Italian better than Google Translate... I've never seen `Tabella dei contenuti` used in in anything serious, it's a literal translation of `Table of contents` and doesn't really mean much in Italian, it's not idiomatic.

Wikipedia itself calls it `Indice dei contenuti`. Also, Google Translate offers two translations for me: `Sommario` and `Indice dei contenuti`, not `Tabella dei contenuti`. DeepL says `Indice dei contenuti`. Finally, in Italian we never use capital letters for common nouns, like in `Contenuti`, it's considered an error and there are no exceptions.

Reply:
sure, understood, will push this change, my bad

Reply:
Thanks! 😃

Reply:
Done https://github.com/adityatelange/hugo-PaperMod/commit/d67462d0f3f22cc5687f73f1edd454aa9cfa3800